tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -553,7 +553,6 @@ def test_positive_virt_card(session, target_sat, module_location, module_org, vm
 
 
 @pytest.mark.e2e
-@pytest.mark.on_premises_provisioning
```

Comment:
why removing on_prem marker?

Reply:
It was not fetching the test case in the PRT, so I removed it but have added it again. I will now run the correct PRT.

Reply:
we use `provisioning: true` for on_prem tests collection